knack_load_fees.py

```diff
@@ -170,6 +173,9 @@ def main():
             raise e
         logger.info(f"Reactivated Knack row id {row['id']}")
 
+    if errors:
+        raise Exception(f"Script completed with {len(errors)} errors.")
+
```

Comment:
so the script is still going to throw an Airflow error every day until we fix the query.....

Reply:
I know that [#10447](https://github.com/cityofaustin/atd-data-tech/issues/10447) is on-going, and it should resolve the dup query in Amanda, but ... I'm sorta torn on if we should quiet this warning to keep the signal-to-noise up in the airflow channel, or let it be and focus on the Amanda problem. I don't know what is the right call, so I am inclined to go with your instinct -- I think your choice to let it complain is a fine one.